_static/kendrasearchtools.js

```diff
@@ -670,19 +678,20 @@ var KendraSearch = {
   /**
    * perform a search for something (or wait until index is loaded)
    */
-   performSearch : function(query) {
+   performSearch : function(query, filters) {
     // create the required interface elements
     this.out = $('#search-results');
     this.title = $('<h2>' + _('Searching...') + '</h2>').appendTo(this.out);
     this.dots = $('<span></span>').appendTo(this.title);
     this.status = $('<p class="search-summary">&nbsp;</p>').appendTo(this.out);
+    // this.searchFilter = $('<div class="search-filter"></div>').appendTo(this.out)
```

Comment:
Can you delete this, or does it serve a purpose?

Reply:
Oh yes, I have deleted it